src/main/resources/format-pages.txt

```diff
@@ -728,6 +728,18 @@ notes = Applications that support .dm3 files include `Datasqueeze <https://www.s
 \n
 Note that the Gatan Reader does not currently support stacks.
 
+[GE MicroCT]
+extensions = .vff
+developer = `GE <http://gelifesciences.com/>`_
```

Comment:
could use `https`

Reply:
I had copied this from one of the other GE formats; b1d59d3 changes all GE links to use ```https``` for consistency.